coderabbit.py

```
@@ -0,0 +1,168 @@
+
+        print("\nReports pushed to MinIO")
+
+    except ResponseError as err:
+        print(f"MinIO Error: {err}")
+
+def deduce_sensitive_data_in_databases():
+    # Initialize config variable
+    config = ConfigParser()
+
+    # If environment variables are not set, read from db.properties file
+    if not all([os.environ.get('db-server'), os.environ.get('db-port'), os.environ.get('db-su-user'),
```

Ruff (0.12.2)

Use capitalized environment variable `DB-SERVER` instead of `db-server`

(SIM112)